src/vs/workbench/services/issue/electron-sandbox/issueService.ts

```diff
@@ -82,6 +88,37 @@ export class NativeIssueService implements IWorkbenchIssueService {
 			const result = [this._providers.has(extensionId.toLowerCase()), this._handlers.has(extensionId.toLowerCase())];
 			ipcRenderer.send('vscode:triggerReporterStatusResponse', result);
 		});
+		ipcRenderer.on('vscode:triggerReporterMenu', async (event, arg) => {
+			if (!this.isRunning) {
+				const extensionId = arg.extensionId;
+
+				// creates menu from contributed
+				const menu = this.menuService.createMenu(MenuId.IssueReporter, this.contextKeyService);
+
+				// render menu and dispose
+				const actions = menu.getActions({ renderShortTitle: true }).flatMap(entry => entry[1]);
+				actions.forEach(async action => {
+					try {
+						if (action.item && 'source' in action.item && action.item.source?.id === extensionId) {
+							this.extensionIdentifierSet.add(extensionId);
+							this.isRunning = true;
+							await action.run();
+						}
+					} catch (error) {
+						console.error(error);
+					} finally {
```

Comment:
maybe here do `this.extensionIdentifierSet.delete(extensionId);` and then you don't need isRunning at all I think